lalrpop-docgen/src/railroad.rs

```diff
@@ -245,7 +245,7 @@ impl LalrpopToRailroad {
         );
 
         let diagram_ref = format!(
-            r#"<img src="{}svg/{}.svg" alt="{}" width="{}" height="{}"/>"#,
+            "![{}svg/{}.svg]({})",
```

Comment:
This was the original choice, but not all rendering engines will use the width/height information in the svg to render the images correctly - and this makes the resulting documents visually sub optimal visually. We could add a flag to switch this behaviour for specific rendering engines though with this as a `generic` default

Reply:
Yes neither choices are right, some eingines will trip up with one, others with the other. This PR works with the engine we are using right now, while the current code causes problems. The .svg images do include the correct dimensions, so the forced with/height is mostly working around bugs in other engines. It feels odd to bias for bugs in engines we don't use and causing issues with the engine we use in return.